service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -168,4 +179,57 @@ public void inviteToTeam(JWTData userData, InviteMembersRequest inviteMembersReq
       emailer.sendInviteEmail(email, "Team Member", inviter, inviterTeam);
     });
   }
+
+  @Override
+  public GetAllTeamsResponse getAllTeams() {
+    List<Team> teamPojos = db.selectFrom(TEAM).fetchInto(Team.class);
```

Comment:
Can you add a sort constraint so that we're returning teams in alphabetical order?